Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_smorci/src/main.rs

```diff
@@ -0,0 +1,72 @@
+use num_bigint::{BigUint, ParseBigIntError};
+use std::io;
+use std::str::FromStr;
+
+const NUMBER_OF_DAYS_IN_A_YEAR: u32 = 365;
+
+fn convert_years_to_days(years: BigUint) -> BigUint {
+    NUMBER_OF_DAYS_IN_A_YEAR * years
+}
+
+fn convert_string_to_biguint(string: &str) -> Result<BigUint, ParseBigIntError> {
+    let result = BigUint::from_str(string.trim())?;
+
+    Ok(result)
```

Comment:
```suggestion
    BigUint::from_str(string.trim())
```
The `let` binding is unnecessary here, just return the result.
(Also it's a little confusing that result is not a Result, meh.)